src/main/interface/msp_box.c

```diff
@@ -278,8 +278,9 @@ int packFlightModeFlags(boxBitmask_t *mspFlightModeFlags)
     memset(mspFlightModeFlags, 0, sizeof(boxBitmask_t));
 
     // enabled BOXes, bits indexed by boxId_e
-    boxBitmask_t boxEnabledMask;
+    boxBitmask_t boxEnabledMask, boxFlightModeMask;
```

Comment:
Please declare each item on a separate line.

Reply:
and done

Reply:
@martinbudden: Never seen that one mentioned as a coding standard for Betaflight, but 100% on board with it!